### DISCUSSION TITLE
Dockerfile打包项目问题？

### DISCUSSION THREAD

**Discussion by gitHubDingGuo** [ChatGPTNextWeb/NextChat#1872] [Q&A]
我看了一下项目下面的dockerfile 文件   开头有一个命令   FROM node:18-alpine AS base   请问这个镜像是直接拉取的hub.docker 还是自己本地自行构建的，如果是本地构建的，能提供一下沟通的文件内容吗。

**Comment by yswlww** [comment]
請更新一下docker的版本謝謝
